AirCasting/Notes/EditNote/EditNoteViewModel.swift

```diff
@@ -33,7 +33,9 @@ class EditNoteViewModelDefault: EditNoteViewModel, ObservableObject {
     }
     
     func deleteTapped() {
-        notesHandler.deleteNote(note: note); exitRoute()
+        notesHandler.deleteNote(note: note, completion: {
+            self.exitRoute()
```

Comment:
`[weak self]` would be better since the user might exit the screen before delete operation is finished

Reply:
sure! Changing